_data/scala-core.yml

```diff
@@ -0,0 +1,78 @@
+# - name: Core Team
+#   description: Knows everything
+#   image: /resources/img/frontpage/scala-spiral.png
+#   city: Lausanne, Switzerland
+#   email: scala.center@epfl.ch
+#   twitter: scala_lang
+#   github: scala
+#   linkedin: https://www.linkedin.com/in/janedoe/
+#   mastodon: https://fosstodon.org/@janedoe
+#   meetup: https://www.meetup.com/your/url/here
+
+- name: Martin Odersky
+  description: Language Creator
+  image: /scala-core/martin.png
+  city: Lausanne, Switzerland
+  github: odersky
+  twitter: odersky
+  linkedin: https://www.linkedin.com/in/sethtisue/
```

Comment:
This is not Martin's linkedin...

Reply:
I thought I checked everything, thanks for noticing!